src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
I was thinking about `react-virtualized` too, but it wouldn't help much, at least for the extension where `maxAge` is `50` by default, and, anyway, we shouldn't hide (virtualize) the new added action rows. The perf waste is due to creating new elements for every dispatched action. This is the part where React is very slow, and we should also take into consideration that unlike the extension, for vanilla Redux DevTools, React is in dev mode, making it even slower. Anyway, Inspector is at least twice faster than LogMonitor.

I added recently `autoincrement` button to [the demo](http://zalmoxisus.github.io/examples/counter/) and it was obviously crashing the extension. Surprisingly it was mostly due to React, not to serialization. Just bouncing the data sent to the extension [solved the problem](https://github.com/zalmoxisus/redux-devtools-extension/releases/tag/v2.9.0).

A benchmark would be rather useful indeed, so we could do it for every new feature to see the cost it adds.